resources/xcode.rb

```diff
@@ -1,5 +1,3 @@
-include Xcode::Helper
-
 resource_name :xcode
 default_action %i(setup install_xcode install_simulators)
```

Comment:
The latest simulators are always installed with Xcode. What's the reasoning behind making `n-1` the default behavior of the resource?

Reply:
That's kind of outside the scope of this change - but the behavior is technically determined by the node attribute `default['macos']['xcode']['simulator']['major_version'] = %w(11 10)`. I can move that attribute to the testing cookbook at some point if you'd prefer.

Reply:
Indeed - it's outside the scope. A separate PR should be opened for that change.